src/components/Todo/AddNew.jsx

```diff
@@ -1,11 +1,19 @@
 import './style.css'
 
-function AddNew() {
+function AddNew({ onAddNew }) {
+
+  const handleFormSubmission = (e) => {
+    e.preventDefault()
+
+    onAddNew( document.getElementById('myInput').value )
```

Comment:
nice, id selector works, you can also use a `useState` to keep track of the value of that input